pipeline_steps/kibana.groovy

```diff
@@ -15,7 +15,7 @@ def kibana(branch, vm=null){
 
 def kibana_prep(branch){
   dir("kibana-selenium") {
-    git url: "https://github.com/rcbops-qe/kibana-selenium.git", branch: "${branch}"
+    git url: env.KIBANA_SELENIUM_REPO, branch: "${branch}"
```

Comment:
Updated to allow branch to be specified as a param, build to test: https://rpc.jenkins.cit.rackspace.net/job/RPC-AIO_mitaka-ceph-periodic/73/

@alextricity25 do you have a runbook of sorts when cutting a new branch on rpc-openstack?  If so, can you update it to ensure that prior to cutting a release that https://github.com/rcbops-qe/kibana-selenium gets a new branch cut as well?